front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -53,158 +60,194 @@ const useScenarioData = (scenario: ScenarioResponse, infra: InfraWithState) => {
       }
     );
 
-  const trainSchedulesResults = useMemo(
-    () => fetchedTrainSchedulesResults || [],
+  const { data: fetchedPacedTrains } =
+    osrdEditoastApi.endpoints.getAllTimetableByIdPacedTrains.useQuery(
+      { timetableId: scenario?.timetable_id },
+      {
+        skip: !scenario || !showPacedTrains,
+      }
+    );
+
+  const formattedRawTrainSchedules = useMemo(
+    () =>
+      (fetchedTrainSchedulesResults || []).map((trainSchedule) => ({
+        ...trainSchedule,
+        id: formatEditoastTrainIdToTrainScheduleId(trainSchedule.id),
+      })),
     [fetchedTrainSchedulesResults]
   );
 
-  const { trainScheduleSummariesById, setTrainScheduleSummariesById, allTrainsLoaded } =
-    useLazyLoadTrains({
+  const formattedRawPacedTrains: PacedTrainResultWithPacedTrainId[] = useMemo(
+    () =>
+      showPacedTrains
+        ? (fetchedPacedTrains || []).map((pacedTrain) => ({
+            ...pacedTrain,
+            id: formatEditoastTrainIdToPacedTrainId(pacedTrain.id),
+          }))
+        : [],
+    [showPacedTrains, fetchedPacedTrains]
+  );
+
+  // TODO Paced trains : pass setTimetableItemIdsToProject in https://github.com/OpenRailAssociation/osrd/issues/10613
+  const { timetableItemSummariesById, setTimetableItemSummariesById, allTimetableItemsLoaded } =
+    useLazyLoadTimetableItems({
       infraId: scenario.infra_id,
       electricalProfileSetId,
-      trainIdsToFetch,
-      trainSchedules,
-      setTrainIdsToProject,
+      timetableItemIdsToFetch,
+      timetableItems,
     });
 
   useEffect(() => {
-    if (allTrainsLoaded) {
-      setTrainIdsToFetch([]);
+    if (allTimetableItemsLoaded) {
+      setTimetableItemIdsToFetch([]);
     }
-  }, [allTrainsLoaded]);
+  }, [allTimetableItemsLoaded]);
 
+  // TODO Paced trains : remove this hook in https://github.com/OpenRailAssociation/osrd/issues/10791
```

Comment:
Maybe it's better like this ?
```suggestion
  // TODO Paced trains : adapt this hook in https://github.com/OpenRailAssociation/osrd/issues/10791
```

Reply:
Done